src/main/java/io/jenkins/plugins/checks/github/GitHubChecksContext.java

```diff
@@ -68,6 +106,60 @@ public GitHubAppCredentials getCredentials() {
      * @return the URL of the summary page
      */
     public String getURL() {
-        return url;
+        if (run == null) {
```

Comment:
Should this have a try catch and then either check for null or be optional?

Reply:
Why do we need to use try-catch, what exceptions will be thrown here?

Reply:
It was about the issue the user hit with the display url not being configured, although Jessie has posted on there saying it’s an invalid configuration so not really an issue I think...

Reply:
the issue is not caused here, in warnings-plugin, we provided the details URL, so it won't call this method here at all.